recipes/ogdf/pr-59-license-files.diff

```diff
@@ -0,0 +1,412 @@
+From 4a3bae804c177f2c63ca097341390e0cb5ab78c3 Mon Sep 17 00:00:00 2001
+From: Hadrien Mary <hadrien.mary@gmail.com>
+Date: Mon, 31 Aug 2020 19:24:13 -0400
+Subject: [PATCH] add license files
+
+---
+ license-files/abacus/LICENSE.txt        | 165 ++++++++++++++++++++++++
+ license-files/backward/LICENSE.txt      |   7 +
+ license-files/bandit/LICENSE.txt        |  21 +++
+ license-files/coin/LICENSE.txt          |  86 ++++++++++++
+ license-files/minisat/LICENSE           |  21 +++
+ license-files/pugixml/LICENSE           |   7 +
+ license-files/snowhouse/LICENSE_1_0.txt |  23 ++++
+ license-files/tinydir/LICENSE           |   9 ++
+ 8 files changed, 339 insertions(+)
+ create mode 100644 license-files/abacus/LICENSE.txt
+ create mode 100644 license-files/backward/LICENSE.txt
+ create mode 100644 license-files/bandit/LICENSE.txt
+ create mode 100644 license-files/coin/LICENSE.txt
+ create mode 100644 license-files/minisat/LICENSE
+ create mode 100644 license-files/pugixml/LICENSE
+ create mode 100644 license-files/snowhouse/LICENSE_1_0.txt
+ create mode 100644 license-files/tinydir/LICENSE
+
+diff --git a/license-files/abacus/LICENSE.txt b/license-files/abacus/LICENSE.txt
+new file mode 100644
+index 00000000..0a041280
+--- /dev/null
++++ b/license-files/abacus/LICENSE.txt
```

Comment:
The license file you could just add a totally separate file in the recipe dir

Reply:
This is what was being done. Idk why it was changed to a patch.

Reply:
Will revert. It was because I couldn't get the correct paths in `license_files` but maybe just adding the folder in the recipe directory is enough?

Reply:
That should work. If not just remove the folder and put them out in the recipe dir right next to the meta.yaml